setup.py

```diff
@@ -57,7 +57,7 @@
     long_description_content_type="text/markdown",
     author="IGES",
     maintainer="Matteo Kimura, Prachi Ingle, Pratham Babaria",
-    maintainer_email="mateus.sakata@gmail.com, phna14@gmail.com, prathambabaria1@gmail.com",
+    maintainer_email="mateus.sakata@gmail.com, prachi.ingle@utexas.edu, prathambabaria1@gmail.com",
```

Comment:
Can you do this with another PR (and also maybe take out Pratham since he's not working on this), it's not really related to the photo stuff?

Reply:
done!